run_tests.py

```diff
@@ -46,16 +46,15 @@ def gdb_inspect(interactive):
                 '--nh', '--nx', 'mimiker.elf']
     gdb = pexpect.spawn(gdb_command, gdb_opts, timeout=3)
     gdb.expect_exact('(gdb)', timeout=5)
-    send_command(gdb, 'klog')
-    send_command(gdb, 'info registers')
-    send_command(gdb, 'backtrace')
-    send_command(gdb, 'list')
-    send_command(gdb, 'kthread')
+    for cmd in commands:
+        send_command(gdb, cmd)
+
     if interactive:
+        print('(gdb)', end='', flush=True)
         gdb.interact()
 
```

Comment:
I think it's much better to put all comands from removed lines 49-53 into `else` clause here.